packages/monaco-editor-wrapper/build/src/buildAssets.mts

```diff
@@ -0,0 +1,10 @@
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+/**
+ * Solves: __dirname is not defined in ES module scope
+ */
+export function getLocalDirectory() {
+    const __filename = fileURLToPath(import.meta.url);
+    return dirname(__filename);
+}
```

Comment:
Is this file supposed to be `.mts`?

Reply:
"Mike Tyson Script" the brother of "Michael Jackson Script" 😆

Reply:
It will leave it as is for now.